apps/site-nft/src/components/MainNavigation/styles.module.less

```diff
@@ -0,0 +1,33 @@
+@import "~@thecointech/site-semantic-theme/variables.less";
+
+.background {
+  background-color: white;
+  grid-row-start: header;
+  grid-row-end: header;
+  grid-column-start: lgap;
+  grid-column-end: rgap;
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
```

Comment:
Here, we had to get a little more specific with the grid, saying precisely where we start and where we end.

However, this is fine, because with this we still ensure that we are in the right place, and that our content (the menu itself) is aligned with the content.